guides/v2.0/coding-standards/technical-vision/index.md

```diff
@@ -5,4 +5,9 @@ title: Magento technical vision
 
 The Magento technical vision is a collection of documents that describe the desired state of the Magento platform.
 
-Each individual technical vision document relates to a set of modules that arelogically grouped together. Individual documents typically describe a component's place in the system, its architecture, extension scenarios. and a list of invariants that must be preserved at all times during component refactoring or extension to ensure consistency.
+Each individual technical vision document relates to a set of modules that are logically grouped together. Individual documents typically describe a component's place in the system, its architecture, extension scenarios. and a list of invariants that must be preserved at all times during component refactoring or extension to ensure consistency.
+
+Technical vision documents are targeting architects and developers to guide them in making decisions based on high-level picture of the system in its desired state.
+
+In contrast, specifications are documents that describe current implementation of the system.
+Though they are also targeting architects and developers, they help with initial investigation, but may be insufficient to make decisions that improve the system.
```

Comment:
Would it make sense to create a list of things technical vision should contain? For instance, architectural principles that current design follows, principles we want to use in the future (desired state), best practices, extension points. Anything else?

Reply:
Technical vision should cover desired state.
It probably makes sense to introduce a template or list some things that are common for a technical vision, I agree. I propose to merge this update as is. I created a separate issue to cover this question https://github.com/magento/architecture/issues/82